tmk_core/chibios.mk

```diff
@@ -233,6 +233,24 @@ qmk: $(BUILD_DIR)/$(TARGET).bin
 dfu-util: $(BUILD_DIR)/$(TARGET).bin cpfirmware sizeafter
 	$(DFU_UTIL) $(DFU_ARGS) -D $(BUILD_DIR)/$(TARGET).bin
 
+
+ifneq ($(strip $(TIME_DELAY)),)
+  TIME_DELAY = $(strip $(TIME_DELAY))
```

Comment:
Only issue here is that if you specify a non-number value, it basically assumes 0, and immediately flashes.

I have no idea now to check for numbers only.

Reply:
Okay. :)

Somebody mentioned using grep to check, but that's waaay beyond me.